python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -469,41 +513,46 @@ def run(self, spectrumSet, detectorMap, seed=1):
             display = afwDisplay.Display(self.debugInfo.arc_frame)
             display.erase()
 
+
+        lineDatas = []
+
         solutions = []
+        
         for spec in spectrumSet:
-            wavelengthCorr = self.fitWavelengthSolution(spec, detectorMap, rng)
+            results = self.fitWavelengthSolution(spec, detectorMap, rng)
+            wavelengthCorr =results[1]
+            lines =results[0]
             if self.debugInfo.display:
                 self.plot(spec, detectorMap, wavelengthCorr)
             solutions.append(wavelengthCorr)
-
-        wlFitData = WavelengthFitData.fromSpectrumSet(spectrumSet, detectorMap)
-        self.measureStatistics(spectrumSet, detectorMap)
-
+            lineDatas.extend(lines)
+            
+        wlFitData = WavelengthFitData(lineDatas)    
+        self.measureStatistics(spectrumSet, detectorMap)        
+        
         return pipeBase.Struct(solutions=solutions, wlFitData=wlFitData)
+        
+    def robustStdev(array):
```

Comment:
This is supposed to be a free function, not a method (note there's no `self` parameter). Looks like I put it in the wrong place, but it doesn't belong here. Put it at the end of the file.